config/build/rollup.config.js

```diff
@@ -276,28 +282,61 @@ const getPackageInfo = (commandLineArgs, pkgName) => {
     const formats = getFormats(commandLineArgs);
 
     const configs = [];
-    if (formats.has("cjs")) {
-        configs.push({
-            name: pkgName,
-            fullName: pkgJson.name,
-            version: pkgJson.version,
-            format: "cjs",
-            platform: "browser",
-            file: pkgJson.main,
-            plugins: [],
-        });
-    }
-    if (formats.has("esm")) {
-        configs.push({
-            name: pkgName,
-            fullName: pkgJson.name,
-            version: pkgJson.version,
-            format: "esm",
-            platform: "browser",
-            file: pkgJson.module,
-            // We care about the file size of this one.
-            plugins: [filesize()],
-        });
+
+    if (pkgJson.exports) {
+        for (const exportConfig of Object.values(pkgJson.exports)) {
+            if (exportConfig.require && formats.has("cjs")) {
```

Comment:
Unrelated to this PR, but we are _extremely_ close to not needing `cjs` builds anymore (in fact, I think we can drop them any time we have bandwidth). :)